linera-storage-service/src/client.rs

```diff
@@ -59,7 +59,7 @@ const MAX_KEY_SIZE: usize = 1000000;
 // is stored to indicate the existence of a namespace.
 #[derive(Clone)]
 pub struct ServiceStoreClientInternal {
-    client: Arc<RwLock<StoreProcessorClient<Channel>>>,
```

Comment:
Is `StoreProcessorClient` directly clonable?

Reply:
No, that is the chosen design of `tonic`. What is Clonable is the `Channel`, not the `StoreProcessorClient` and every operation is `&mut ` which prevents sharing.